pkg/types/types.go

```diff
@@ -124,6 +134,7 @@ func (c *IPConfig) MarshalJSON() ([]byte, error) {
 	return json.Marshal(ipc)
 }
 
+// UnmarshalJSON parses the JSON-encoded data to IPConfig
```

Comment:
Probably:
`UnmarshalJSON parses the JSON-encoded data and stores the result in the value pointed to by c.`
You could use this template for all of the UnmarshalJSON docstrings.